topic_folders/policies/immediate-response-checklist.md

```diff
@@ -0,0 +1,74 @@
+- [ ] LastPass
+- [ ] AMY
+- [ ] Regional Email
+- [ ] Asana
+- [ ] HelpScout
+- [ ] Zoom
+- [ ] Workshop Administrator’s Slack Channel
+- [ ] Workshop Administrator’s Email List
+
+### Instructor Training
+The suspended member will, for the duration of their suspension, be removed from the “Our Trainers” web page. Additionally, the suspended member will not be allowed to: 
+- [ ] Teach Instructor Training or lead teaching demonstration sessions. The Deputy Director of Instructor Training (DDIT) will check the teaching demo Etherpad to ensure that the suspended member is not signed up to lead teaching demonstrations and will find a replacement Trainer as needed. 
+- [ ] Join Trainer meetings. The DDIT will check the meeting Etherpad, and if the suspended member is signed up to attend, will email them if needed to confirm they will not be in attendance. 
+- [ ] Post pull requests (PRs), issues, or comments to the Instructor Training GitHub repository. Write or administrative access to repositories will be removed for the duration of the suspension. 
+- [ ] Post to the Trainers Slack channel or mailing lists. The suspended member will be removed from these lists for the duration of their suspension. 
+- [ ] Participate in review of applications for open instructor training or Trainer training. 
+Participate as a learner in instructor training.
+
+### Trainer Training
+The suspended member will not be allowed to participate in Trainer Training (if they are currently enrolled) and will not be accepted for Trainer Training (if they apply during the suspension period). Additionally, they will not be allowed to shadow a current Carpentries Trainer. They may apply or reapply for future rounds of Trainer Training. 
+
+### Member Organisation and Local Activities
+The suspended member shall not be involved in officially branded Carpentries workshops or activities at the member or local site for the duration of the suspension. We expect the suspended member to assert that they will be able to carry out this suspension accordingly and to notify the CoC committee if this suspension will interfere with their expected employment duties. 
+
+If the suspension does affect employment duties or the suspended member is the key contact for a member site, the member will work with The Carpentries Executive Director to determine a course of action.
+
+### Lesson Development and Maintenance
+The suspended member will not be allowed to post PRs, issues, or comments to any repository in the carpentries, including but not limited to datacarpentry, swcarpentry, library carpentry, data-lessons, and carpentrieslab organizations. Administrative or write privileges to repositories in these organizations will be removed for the duration of the suspension.
+
+If the suspended member is a Maintainer, their name will be removed from the lessons page and they will not appear on the Maintainers page for the duration of their suspension. 
```

Comment:
Again something I think should probably be adjusted to level of seriousness for issues under review.

Reply:
I'm curious how some of these permissions will be set. Will they be banned from the repos via github permissions, or simply instructed not to touch things?

I think it's fine to have language that the CoCc has the right to enforce these with permissions settings, but I'd like to hope/think that most people would be willing to self-regulate to stay away.

So there's a clear right that the committee may choose to, per committee choice: 

*enforce things via system permissions, or
* request that the person stay away, or
* do neither if the committee deems it not applicable or not a problem.

Reply:
Their access to the repos will be revoked. Yes, depends on seriousness of the incident. Language in intro paragraph applies here as well.